documentation/src/components/Sights/TopBar/TopBar.tsx

```diff
@@ -21,8 +21,7 @@ export interface TopBarProps {
   onSelectTab: (tab: Tab) => void;
   onLookupInput: (value: string) => void;
 }
-
-export const tabs = {
+export const tabs: Record<'sights' | 'vehicles' | 'labels', Tab> = {
```

Comment:
Remove this type declaration : let TypeScript infer the type

Reply:
It's inferring the types only while we use the tabs value not putting any restrictions while writing the code.
for example.
```ts
const tabs = {/** the tab value is wrong */} // won't tell error on hear
const a: Tab = tabs.sight; // will tell error.
```

The main problem is if someone adds a new property for one of the tabs and uses it we won't be able to find it on code review. after 10 iterations on the tabs value the things will be out of track.